### PR TITLE
Chem tutorial 2

### PR DESCRIPTION
**Pull request by dchung0741**
Chemistry tutorial involving:
- Hamiltonian generation.
- Fermion-qubit mapping.
- Advanced guide on computing electron integrals.

### CODE REVIEW

File: docs/tutorials/advanced_guide_on_mo_ints.ipynb [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now demonstrate the detailed steps to obtain the `SpinMOeInt` from molecule without the `get_spin_mo_integrals_from_mole` function introduced in the [Hamiltonian generation tutorial](https://quri-parts.qunasys.com/tutorials/mo_eint_and_hamiltonian):\n",
```
Comment (kwkbtr) [inline docs/tutorials/advanced_guide_on_mo_ints.ipynb]:
```suggestion
    "We now demonstrate the detailed steps to obtain the `SpinMOeInt` from molecule without the `get_spin_mo_integrals_from_mole` function introduced in the [Hamiltonian generation tutorial](mo_eint_and_hamiltonian.ipynb):\n",
```

File: docs/tutorials/mo_eint_and_hamiltonian.ipynb [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+    "\tactive_space,jordan_wigner  # Default to jordan wigner.\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `full_space_jw_hamiltonian` and `active_space_jw_hamiltonian` are the hamiltonian we desired. The full space and active space operator and state mapper are functions that are able to perform fermion-qubit mapping for other operators and states in further computations. Their usage can be found in the [Fermion-Qubit Mapping Hamiltonian Tutorial](https://quri-parts.qunasys.com/tutorials/qubit_operator_mapping)."
```
Comment (kwkbtr) [inline docs/tutorials/mo_eint_and_hamiltonian.ipynb]:
```suggestion
    "The `full_space_jw_hamiltonian` and `active_space_jw_hamiltonian` are the hamiltonian we desired. The full space and active space operator and state mapper are functions that are able to perform fermion-qubit mapping for other operators and states in further computations. Their usage can be found in the [Fermion-Qubit Mapping Hamiltonian Tutorial](qubit_operator_mapping.ipynb)."
```

File: docs/tutorials/qubit_operator_mapping.ipynb [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+   "source": [
+    "## Getting the Electron integrals and the Hamiltonian"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We first prepare a fermionic Hamiltonian for later demonstration. For generating the molecular Hamiltonian, please refer to the [Hamiltonian generation tutorial](https://quri-parts.qunasys.com/tutorials/mo_eint_and_hamiltonian)."
```
Comment (kwkbtr) [inline docs/tutorials/qubit_operator_mapping.ipynb]:
```suggestion
    "We first prepare a fermionic Hamiltonian for later demonstration. For generating the molecular Hamiltonian, please refer to the [Hamiltonian generation tutorial](mo_eint_and_hamiltonian.ipynb)."
```

Comment (dchung0741) [reply docs/tutorials/qubit_operator_mapping.ipynb]: I have fixed this.

File: docs/tutorials/mo_eint_and_hamiltonian.ipynb (hunk 2 above)
Comment (dchung0741) [reply docs/tutorials/mo_eint_and_hamiltonian.ipynb]: I have fixed this.

File: docs/tutorials/advanced_guide_on_mo_ints.ipynb (hunk 1 above)
Comment (dchung0741) [reply docs/tutorials/advanced_guide_on_mo_ints.ipynb]: I have fixed this.

**Comment by dchung0741** [comment]
Do we want the nested sidebar for the chemistry tutorials as well?

**Comment by kwkbtr** [comment]
Yeah, that may be a good idea!

**Comment by dchung0741** [comment]
I just added it.

**Review by kwkbtr — APPROVED** [review]
Great, thanks!

**Pull request merged**